src/js/components/form-elements/FilterSelectField.jsx

```diff
@@ -108,41 +108,21 @@ IndicatorsContainer.propTypes = {
 };
 
 const FilterSelectField = (props) => {
-  const renderInput = ({ className, ...attributes }) => {
```

Comment:
here I have rolled back my changes from previous pull requests that was supposed to fix this bug but apparently it didn;t work for some cases https://github.com/openboxes/openboxes/pull/3669
After further investigation I managed to find the exact problem which was having submit buttons in the form.